granian/cli.py

```diff
@@ -194,6 +194,38 @@ def option(*param_decls: str, cls: Optional[Type[click.Option]] = None, **attrs:
     default=False,
     help="Enable auto reload on application's files changes (requires granian[reload] extra)",
 )
+@option(
+    '--reload-paths',
+    type=click.Path(exists=True, file_okay=True, dir_okay=True, readable=True, path_type=pathlib.Path),
+    default=(pathlib.Path.cwd(),),
+    help='Paths to watch for changes',
+    multiple=True,
+)
+@option(
+    '--reload-ignore-dirs',
+    default=(),
+    help=(
+        'Names of directories to ignore (i.e. should not trigger reload). '
+        'Extends the default list of directories to ignore in watchfiles.filters.DefaultFilter.'
+    ),
+    multiple=True,
+)
+@option(
+    '--reload-ignore-entity-patterns',
```

Comment:
I would rename this to `--reload-ignore-patterns`, just for the sake of simplicity

Reply:
Renamed everywhere, internally as well.